### DISCUSSION TITLE
update?

### DISCUSSION THREAD

**Discussion by Wierzciu** [kdalanon/ChatGPT-AutoHotkey-Utility#6] [General]
Hello,
did You plan update script? currently dont work with chatgpt/openai api. Use macOS on private computer and have www.elephas.app but on working machine use windows, looking for  alternative but only see script for autohotkey and all dont work with currently chatgpt/openai api.
Will be greatful for information 😃

**Comment by michaelbeijer** [comment]
Hmm, it's working for me with the ChatGPT/OpenAI api. I'm on Windows.

**Reply by Wierzciu** [reply]
Hello,
sorry, You have right, I have some issues with my API settings. I have another question, it's possible to don't show window with copy and paste instantly text under the orginal one? 😃

**Comment by michaelbeijer** [comment]
I have integrated it into my own app: https://beijer.bot/
